next.config.mjs

```diff
@@ -35,6 +35,12 @@ const nextConfig = {
           "https://join.slack.com/t/prairielearn/shared_invite/zt-13kx0hg6v-uuC3kyt_3iBxjSpyhCbYVw",
         permanent: true,
       },
+      {
+        source: "/sigcse2024",
+        destination:
+          "/blog/sigcse2024_AffiliatedEvent",
```

Comment:
Given that this isn't using any kind of blog architecture, is there a reason to not just serve the page directly from `/sigcse2024`?

Reply:
I thought this would be easier, since we will (hopefully soon) get the blog page up.

So your suggestion is to add text at `pages/sigcse2024` and then there is no need to redirect.

After SIGCSE, we move the affiliated event text to `pages/blog/sigcse2024_AffiliatedEvent` (since we will post news at `pages/blog/sigcse2024`) and use the redirect.

Let me know if I am understanding this correctly.